features/step_definitions/common_steps.rb

```diff
@@ -0,0 +1,87 @@
+Given /^I am on (.+)$/ do |page_name|
+  visit path_to(page_name)
+end
+
+When /^I go to (.+)$/ do |page_name|
+  visit path_to(page_name)
+end
+
+When /^I press "([^\"]*)"$/ do |button|
+  click_button(button)
+end
+
+When /^I click "([^\"]*)"$/ do |link|
+  click_link(link)
+end
+
+When /^I fill in "([^\"]*)" with "([^\"]*)"$/ do |field, value|
+  fill_in(field.gsub(' ', '_'), :with => value)
+end
+
+When /^I fill in "([^\"]*)" for "([^\"]*)"$/ do |value, field|
+  fill_in(field.gsub(' ', '_'), :with => value)
+end
+
+When /^I fill in the following:$/ do |fields|
+  fields.rows_hash.each do |name, value|
+    When %{I fill in "#{name}" with "#{value}"}
+  end
+end
+
+When /^I select "([^\"]*)" from "([^\"]*)"$/ do |value, field|
+  select(value, :from => field)
+end
+
+When /^I check "([^\"]*)"$/ do |field|
+  check(field)
+end
+
+When /^I uncheck "([^\"]*)"$/ do |field|
+  uncheck(field)
+end
+
+When /^I choose "([^\"]*)"$/ do |field|
+  choose(field)
+end
+
+Then /^I should see "([^\"]*)"$/ do |text|
+  page.should have_content(text)
+end
+
+Then /^I should see \/([^\/]*)\/$/ do |regexp|
+  regexp = Regexp.new(regexp)
```

Comment:
I resued steps from JustReadIt! project, I don't remember why I wrote that then. I will look into it.